.github/workflows/build.yml

```diff
@@ -38,10 +38,34 @@ jobs:
       - uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          files: ./plugin/coverage/clover.xml,./providers/deepl/coverage/clover.xml
+          files: ./plugin/coverage/clover.xml,./providers/deepl/coverage/clover.xml,./providers/libretranslate/coverage/clover.xml
           flags: unittests
           name: codecov-umbrella
 
+  e2e:
+    strategy:
+      matrix:
+        node: [14, 16, 18]
+        strapi: [4.6.2, 4.7.1, 4.8.2]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'yarn'
+      - run: cp playground/.env.test playground/.env
```

Comment:
I think I would prefer if we skip this step and instead have something like `   "e2e": "ENV_PATH='./.env.test' start-server-and-test 'yarn start' http://0.0.0.0:1337 'yarn cy:run'"` in the package.json file